### ISSUE TITLE
Fix ReqMgrAux postCampaignConfig API

### ISSUE THREAD

**Issue by vkuznet** [dmwm/WMCore#9233]
When I invoke the following code:
```
from WMCore.Services.ReqMgrAux.ReqMgrAux import ReqMgrAux
mgr = ReqMgrAux(url, header=header, logger=logger)
mgr.postCampaignConfig(campaign, config)
```
it produces the following error:
```
httplib.HTTPException: url=https://vk-reqmgr.cern.ch/reqmgr2/data/data/campaignconfig/HINPbPbSpring18DR, code=405, reason=Method Not Allowed, headers={'Content-Length': '766', 'X-Error-Http': '405', 'Vary': 'Accept-Encoding', 'X-Rest-Time': '1501.083 us', 'Server': 'CherryPy/unknown', 'X-Error-Detail': 'API not supported for this request method', 'Allow': '', 'X-Rest-Status': '204', 'Date': 'Wed, 29 May 2019 20:07:14 GMT', 'Content-Type': 'text/html;charset=utf-8', 'CMS-Server-Time': 'D=20145 t=1559160434234088', 'X-Error-Id': '0c87d84b60cd9c769a19d4454f99cfb0'}
```
According to ReqMgr log the traceback shows that such method is not supported
```
[29/May/2019:22:07:14]  SERVER REST ERROR WMCore.REST.Error.APIMethodMismatch 0c87d84b60cd9c769a19d4454f99cfb0 (API not supported for this request method)
[29/May/2019:22:07:14]    Traceback (most recent call last):
[29/May/2019:22:07:14]      File "/data/srv/HG1904a/sw/slc7_amd64_gcc630/cms/reqmgr2/1.2.1.pre2/lib/python2.7/site-packages/WMCore/REST/Server.py", line 711, in default
[29/May/2019:22:07:14]        return self._call(RESTArgs(list(args), kwargs))
[29/May/2019:22:07:14]      File "/data/srv/HG1904a/sw/slc7_amd64_gcc630/cms/reqmgr2/1.2.1.pre2/lib/python2.7/site-packages/WMCore/REST/Server.py", line 760, in _call
[29/May/2019:22:07:14]        raise APIMethodMismatch()
[29/May/2019:22:07:14]    APIMethodMismatch: APIMethodMismatch 0c87d84b60cd9c769a19d4454f99cfb0 [HTTP 405, APP 204, MSG 'API not supported for this request method', INFO None, ERR None]
```

**Comment by vkuznet** [comment]
ReqMgrAux initialization was done by reading plenty of code to figure out proper set of parameters, see https://github.com/dmwm/WMCore/issues/9234

**Comment by vkuznet** [comment]
Turns out the error cause by wrong URL, but WMCore should be more verbose to tell what happens. The fix is in #9235

**Comment by vkuznet** [comment]
The wrong url was `https://vk-reqmgr.cern.ch/reqmgr2/data` while it should be `https://vk-reqmgr.cern.ch/reqmgr2`. The extra `/data` was treated as API which was not supported. The error message is obscure and does not tell exactly what was the problem, i.e. which API it complains. That's why I opened up #9235

**Comment by amaltaro** [comment]
If you have proposed a fix for it, then I think we can leave it opened until that PR gets merged.

**Comment by amaltaro** [comment]
Fixed by #9235 9235
